style.css

```diff
@@ -0,0 +1,386 @@
+* {
+	box-sizing: border-box;
+	margin: 0;
+	font-size: 10px;
+	font-family: Ubuntu, Sans-serif;
+	color: white;
+	font-weight: bold;
+}
+br {
+	display:none;
+}
+body {
+	background-color: #C06C84 ;
+	width: 100%;
+	font-family: "Idim";
+}
+nav {
+	width: 100%;
+	display: flex;
+	justify-content: space-between;
+	height: 50px;
+	position: fixed;
+	padding: 15px;
+	background-color: #C06C84;
+}
+
+footer {
+	text-align: center;
+	background-color: #355C7D;
+	padding-bottom: 30px;
+}
+
+a:visited {
+	color: inherit;
+}
+
+
+
+.nav-left-container {
+	display: flex;
+	justify-content: flex-start;
+	gap: 30px;
+	color: white;
+}
+
+.nav-left-container a {
+	text-decoration: none;
+	font-size: 2rem;
+	color: white;
+}
+
+nav > a {
+	font-size: 2rem;
+	text-decoration:none;
+}
+
+.intro {
+	display: flex;
+	justify-content: center;
+	flex-wrap: wrap-reverse;
+	padding: 15px;
+	align-items: center;
+	max-width: 1000px;
+	margin: 0 auto;
+	height: 100%;
+	gap: 30px;
+	
+
+	
+}
+
+.intro > img {
+	width: 300px;
+	object-fit: cover;
+	border-radius: 5px;
+}
+.intro-texts > h2 {
+	font-size: 3.5rem;
+	letter-spacing: 3px;
+	line-height: 80px;
+}
+.intro-texts > h1 {
+	font-size: 7rem;
+	letter-spacing: 1.5px;
+	width: 100%;
+	line-height: 80px;
+	margin: 35px 0;
+}
+
+.intro-texts > h3 {
+	font-size: 2.4rem;
+	letter-spacing: 2px;
+	line-height: 40px;
+}
```

Comment:
for these 4 you have also used the direct child selector, this works but it would stop working if you had to add a layer in between e.g. a div around the image. so i would recommend instead using the normal parent child selector we covered in class

Reply:
OK! Thanks. done